third_party/move/move-compiler-v2/src/lib.rs

```diff
@@ -48,6 +49,8 @@ pub fn run_move_compiler(
     let mut env = run_checker(options.clone())?;
     check_errors(&env, error_writer, "checking errors")?;
 
+    function_checker::check_functions(&mut env);
```

Comment:
It would be nice to have either the module or the method name be more specific.

We have `run_checker`, `check_errors`, `check_functions`, ... Unless I go into the definition and read what is happening, I can bet very few people can even roughly guess what is happening here.

Feel free to ignore this as out of scope for this PR.

Reply:
I renamed the (now 2 ) functions to be more specific.